### DISCUSSION TITLE
Medulla Demos 06/24

### DISCUSSION THREAD

**Discussion by keithlogan94** [recro/medulla#168] [Show and tell]
### Summary

# Medulla Demos

The goal of this ticket is to record what will be demo'd for Medulla Demos 06/24.

<b>Everyone who will demo, should add a comment to this issue detailing what you will be demoing on 06/24.</b>

We will do this in standup as well.

**Comment by keithlogan94** [comment]
<b>Demo By:</b> Keith Becker
<b>Title:</b> Workflow Designer Demo
<b>What will be in the demo:</b> 
A workflow designer for designing actionable scripts where each action has input ports, and output ports. You can draw a line from one output port to any other input port on an action. The Action controller will make sure to route the data, and run the actions.

**Comment by keithlogan94** [comment]
<b>Demo By:</b> Jonathan
<b>Title:</b> Deploying the Flux Helm Chart and the Medulla Helm Chart
<b>What will be in the demo:</b> 
Deploying flux and medulla charts through Kubernetes.

**Comment by andrewbabbittdev** [comment]
Discussions would be a better area for stuff like this, specifically the show and tell category

**Comment by keithlogan94** [comment]
Tried to convert it but it gave an error

Unable to convert this issue to a discussion. Category is not accessible to the actor
